package.json

```diff
@@ -140,6 +140,7 @@
       "prettier"
     ],
     "rules": {
+      "react/react-in-jsx-scope": "off",
```

Comment:
wasthis? Does this allow us to declare JSX files without React imports?

Reply:
its an eslint rule I doubt we'll ever be affected by, i believe it'll warn you off from trying to use react in jsx files if it isn't in scope... probably something you can get away with in theory because of the looser typing?